cluster/manifests/ingress-controller/deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     metadata:
       labels:
         application: kube-ingress-aws-controller
-        version: v0.7.4
+        version: v0.8.0
```

Comment:
this needs an IAM role modification!
https://github.com/zalando-incubator/kube-ingress-aws-controller#v080-to-v080